Source/Scene/Cesium3DTile.js

```diff
@@ -598,10 +690,6 @@ define([
 
         this.replacementNode = undefined;
 
-        // Restore properties set per frame to their defaults
-        this.distanceToCamera = 0;
-        this.visibilityPlaneMask = 0;
-        this.selected = false;
```

Comment:
These aren't needed to be reset anymore since they are always updated right away in the traversal.